customizations/mei-basic.xml

```diff
@@ -53,7 +53,7 @@
             <change n="2" when="2019-10-26" who="#JK">
                 <desc>Revisions following the discussions on GitHub, and at the Nashville Hackathon.</desc>
             </change>
-            <change n="3" when="2023-05-24" who="#LP">
+            <change n="3" when="2023-06-12" who="#LP">
```

Comment:
No need to update this here. If you want the changes to have a common entry, it should be:
```suggestion
            <change n="3" from="2023-05-24" to="2023-06-12" who="#LP">
```